### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Don't merge until https://github.com/dotnet/AspNetCore.Docs/pull/28826 merges 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/deploying/native-aot/index.md](https://github.com/dotnet/docs/blob/a8c85f1dc2d5b64cf606ac9d2ab5f17063384bac/docs/core/deploying/native-aot/index.md) | [docs/core/deploying/native-aot/index](https://review.learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/index?branch=pr-en-us-34841) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
https://learn.microsoft.com/aspnet/core/native-aot/  is now live

@JamesNK can you approve this?

File: docs/core/deploying/native-aot/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ ms.date: 02/21/2023
 
 Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps start up very quickly and use less memory. Users of the application can run it on a machine that doesn't have the .NET runtime installed.
```
Comment (Rick-Anderson) [inline docs/core/deploying/native-aot/index.md:10]:
```suggestion
Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps start up quickly and use less memory. The app can run it on a machine that doesn't have the .NET runtime installed.
```
Per MS sytle guide mantra, less is more.

File: docs/core/deploying/native-aot/index.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ ms.date: 02/21/2023
 ---
 # Native AOT deployment
 
-Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps start up very quickly and use less memory. Users of the application can run it on a machine that doesn't have the .NET runtime installed.
+Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps start up quickly and use less memory. The app can run it on a machine that doesn't have the .NET runtime installed.
```
Comment (am11) [inline docs/core/deploying/native-aot/index.md:10]: Perhaps it's better this way:
```suggestion
Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps have faster startup time and smaller memory footprints. These apps can run on a machine without .NET runtime installed.
```

cc @jkotas

Comment (eerhardt) [reply docs/core/deploying/native-aot/index.md:10]: (nit) `footprints` => `footprint`. Unless you want to make `time` plural as well.

Comment (am11) [reply docs/core/deploying/native-aot/index.md:10]: Update the suggestion, thanks.

Comment (am11) [reply docs/core/deploying/native-aot/index.md:10]: Updated the suggestion, thanks.

File: docs/core/deploying/native-aot/index.md [hunk 3]
```diff
@@ -7,9 +7,9 @@ ms.date: 02/21/2023
 ---
 # Native AOT deployment
 
-Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps start up very quickly and use less memory. Users of the application can run it on a machine that doesn't have the .NET runtime installed.
+Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps have faster startup time and smaller memory footprint. These apps can run on a machine without .NET runtime installed.
```
Comment (Rick-Anderson) [inline docs/core/deploying/native-aot/index.md:10]:
```suggestion
Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps have faster startup time and smaller memory footprints. These apps can run on a machine without .NET runtime installed.
```

File: docs/core/deploying/native-aot/index.md [hunk 4]
```diff
@@ -7,9 +7,9 @@ ms.date: 02/21/2023
 ---
 # Native AOT deployment
 
-Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps start up very quickly and use less memory. Users of the application can run it on a machine that doesn't have the .NET runtime installed.
+Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps have faster startup time and smaller memory footprints. These apps can run on a machine without .NET runtime installed.
```
Comment (Rick-Anderson) [inline docs/core/deploying/native-aot/index.md:10]:
```suggestion
Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps start up quickly and use less memory. The app can run it on a machine that doesn't have the .NET runtime installed.
+Publishing your app as *native AOT* produces an app that is [self-contained](../index.md#publish-self-contained) and that has been ahead-of-time (AOT) compiled to native code. Native AOT apps have faster startup time and smaller memory footprints. These apps can run on a machine without .NET runtime installed.
```

**Comment by mitchdenny** [comment]
Looks good to me with the change proposed.

**Review by JamesNK — APPROVED** [review]

**Pull request merged**